source/fundamentals/authentication.txt

```diff
@@ -388,6 +388,113 @@ see the corresponding syntax for specifying the ``X.509`` authentication mechani
          You can alternatively pass ``null`` as the parameter to prompt the MongoDB
          server to infer the username based on your ``X.509`` certificate.
 
+MONGODB-OIDC
+~~~~~~~~~~~~
+
+.. important::
+
+   The MONGODB-OIDC authentication mechanism requires MongoDB v7.0 or later running
+   on a Linux platform.
+
+The following sections describe how to use the MONGODB-OIDC authentication mechanism to
+authenticate to various platforms.
```

Comment:
Nit: Would "onto" make more sense here? Rather than to?

Reply:
It's probably "from" those platforms, actually